File: arrays/Two_sum/two_sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        vis = {}
        for i in range(len(nums)):
            if  target - nums[i] in vis:
                return [vis[target - nums[i]],i]
            vis[nums[i]] = i

File: arrays/Two_sum/two_sum.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        int size = nums.size();
        unordered_map<int,int> hmap;

        for (int i = 0; i < size; i++){
            if( hmap.find((target - nums[i])) == hmap.end()){
                hmap[nums[i]] = i;
            }
            else {
                return {hmap[target - nums[i]],i};
            }
        }
        return {};
    }
};
